src/JustEat.StatsD/TagsFormatter.cs

```diff
@@ -17,6 +17,11 @@ public static class TagsFormatter
     /// </summary>
     public static IStatsDTagsFormatter DataDog => new TrailingTagsFormatter();
 
+    /// <summary>
+    /// Gets a GraphiteDB tags formatter.
+    /// </summary>
+    public static IStatsDTagsFormatter GraphiteDb => new GraphiteDbTagsFormatter();
```

Comment:
This highlights that we should have probably made these singletons via `{ get; } = ...` rather than creating a new instance on every access.

I'll fix that in a separate PR after this is merged.